.github/workflows/post_to_mastodon.sh

```
@@ -0,0 +1,26 @@
+
+I have a new release in for the Snakemake executor for #SLURM on #HPC systems. The version now is '${version}'.
+
+See ${changelog} for details.
+
+Get the latest release from #Bioconda or #Pypi. Be sure to give it some time to be released there, too.
+
+#OpenScience #ReproducibleResearch #ReproducibleComputing
+
+EndOfText
+
+curl -X POST -H "Authorization: Bearer ${{ secrets.MASTODONBOT }}" \
```

Shellcheck (0.10.0)

[error] Parameter expansions can't start with {. Double check syntax.

(SC2296)